assets/pages/regular-contributions/components/contributionsCheckoutContainer.jsx

```diff
@@ -9,27 +9,19 @@ import ContributionsCheckout from 'components/contributionsCheckout/contribution
 
 import { type PageState as State } from '../regularContributionsReducer';
 import RegularContributionsPayment from './regularContributionsPayment';
-import RegularInlineContributionsPayment from './regularInlineContributionsPayment';
 
 
 // ----- State Maps ----- //
 
 function mapStateToProps(state: State) {
-
-  const inlineCardPaymentVariant = state.common.abParticipations.inlineStripeFlowCardPayment;
-
   return {
     amount: state.page.regularContrib.amount,
     currencyId: state.common.internationalisation.currencyId,
     country: state.common.internationalisation.countryId,
-    inlineCardPaymentVariant,
-    payment: inlineCardPaymentVariant === 'inline' ?
-      <RegularInlineContributionsPayment /> :
-      <RegularContributionsPayment />,
+    payment:  <RegularContributionsPayment />,
```

Comment:
Since there is no variant, this could be push to the `jsx` view. Is that correct @Ap0c ?

Reply:
It is a different element passed through for one off and regular contributions, so this still needs to be passed through by the container

Reply:
True but it doesn't have to live in the container any more, it can be passed through in `regularContributions.jsx` as before. It was only in the container temporarily because we needed to access `inlineCardPaymentVariant`.